app/src/main/java/org/thoughtcrime/securesms/ui/Colors.kt

```diff
@@ -90,6 +90,8 @@ interface Colors {
     val qrCodeBackground: Color
 }
 
+val Colors.slimOutlineButton: Color get() = text
```

Comment:
Why does this extension function exist? If this is always set to text then why not set that the composable at declaration?